### ISSUE TITLE
Specifying a custom port via cli fails with Error

### ISSUE THREAD

**Issue by stfsy** [broccolijs/broccoli#315]
Starting broccoli using the cli with the argument --port and a valid port number fails, because port is a string, but server.serve only accepts numbers.

```
> broccoli serve --host=0.0.0.0 --port=8000

C:\...\broccoli\lib\server.js:11
    if (typeof port !== 'number' || port !== port) throw new Error('Expected port to bind to (e.g. 4200)')
                                                   ^

Error: Expected port to bind to (e.g. 4200)
    at Object.serve (C:\cygwin64\home\Stefan\stfsy.github.io\node_modules\broccoli\lib\server.js:11:58)
    at Command.<anonymous> (C:\cygwin64\home\Stefan\stfsy.github.io\node_modules\broccoli\lib\cli.js:23:29)
    at Command.listener (C:\cygwin64\home\Stefan\stfsy.github.io\node_modules\commander\index.js:301:8)
    at emitTwo (events.js:106:13)
    at Command.emit (events.js:191:7)
    at Command.parseArgs (C:\cygwin64\home\Stefan\stfsy.github.io\node_modules\commander\index.js:615:12)
    at Command.parse (C:\cygwin64\home\Stefan\stfsy.github.io\node_modules\commander\index.js:458:21)
    at Object.broccoliCLI [as cli] (C:\cygwin64\home\Stefan\stfsy.github.io\node_modules\broccoli\lib\cli.js:56:13)
    at Object.<anonymous> (C:\cygwin64\home\Stefan\stfsy.github.io\node_modules\broccoli-cli\bin\broccoli:20:10)
    at Module._compile (module.js:541:32)
```

**Comment by bvbg** [comment]
Is there any workaround for this at the moment - other than hacking the code?

**Comment by stfsy** [comment]
npm i stfsy/broccoli --save-dev ? 😀

**Comment by bvellacott** [comment]
cheers

**Comment by joliss** [comment]
Thanks. Fixed and released in v1.1.1.

**Comment by bvellacott** [comment]
cheers
